arch/risc-v/src/common/riscv_fpu.S

```diff
@@ -74,7 +75,7 @@
 
 riscv_fpuconfig:
     li           a0, FS_INITIAL
-    csrs         mstatus, a0
+    csrs         CSR_STATUS, a0
     csrwi        fcsr, 0
     ret
 
```

Comment:
how about change riscv_savefpu/riscv_restorefpu to macro or merge into save_ctx and restore_ctx